bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/bno/GetMembershipsFlowResponder.kt

```diff
@@ -22,8 +22,6 @@ class GetMembershipListFlowResponder(flowSession : FlowSession) : BusinessNetwor
         //build memberships snapshot
         flowSession.receive<MembershipListRequest>()
         val databaseService = serviceHub.cordaService(DatabaseService::class.java)
-        val memberships = databaseService.getActiveMemberships()
-
```

Comment:
General question about this flow responder:
The description states: " * Responder to the [GetMembershipsFlow]. Only active members can request a membership list."
Where does this function restrict the response only to active members?

Reply:
It's in the parent flow `BusinessNetworkOperatorInitiatedFlow`

Reply:
That's the beauty of the BusinessNetworkOperatorInitiatedFlow and the other BusinessNetworkAware flows